src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentsPageV2.jsx

```diff
@@ -73,20 +98,23 @@ function AppointmentsPageV2({
   const routes = (
     <Switch>
       <Route exact path="/" component={UpcomingAppointmentsList} />
-      <Route path="/requested" component={UpcomingAppointmentsList} />
+      <Route path="/requested" component={RequestedAppointmentsList} />
       <Route path="/past" component={PastAppointmentsList} />
       <Route path="/cancelled" component={UpcomingAppointmentsList} />
     </Switch>
   );
 
-  function onChange(e) {
-    if (e.currentTarget.value === 'upcoming') {
+  function onDropdownChange(e) {
```

Comment:
Doesn't look like this function is covered in tests, might want to add one that switches pages.

Reply:
Added tests for page switches on dropdown change